Tick2q/Tick2q.fsproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
Change to reflect the correct target framework, since this project uses string interpolation features.